cmd/libs/go2idl/generator/import_tracker.go

```diff
@@ -44,48 +49,110 @@ func NewImportTracker(types ...*types.Type) *ImportTracker {
 			// Add other forbidden keywords that also happen to be
 			// package names here.
 		},
+		lang: "golang",
 	}
 	tracker.AddTypes(types...)
 	return tracker
 }
 
+func NewProtobufImportTracker(local types.Name, types ...*types.Type) *ImportTracker {
```

Comment:
Should this be in the core go2idl?  I would assume that only golang would be in go2idl, and protobuf would be in a separate package outside of go2idl (i.e., protobuf composes go2idl, which doesn't know anything except golang).